windows/src/developer/TIKE/child/UfrmEditor.pas

```diff
@@ -200,6 +200,9 @@ procedure TfrmEditor.UpdateEditorFormat;
   else if Pos('htm', s) > 0 then EditorFormat := efHTML
   else if Copy(s,1,4) = '.php' then EditorFormat := efHTML
   else if Pos('xml', s) > 0 then EditorFormat := efXML
+  else if s = '.json' then EditorFormat := efJSON
```

Comment:
Do you also want the other Keyman file extensions that use an "XML" (`kvks`) / "JSON" (`keyman-touch-layout`) format?

Reply:
They are handled by the keyboard editor already, so at this stage, I don't think it's necessary.